src/views/wishlist.pug

```diff
@@ -32,13 +32,16 @@ block content
         +sharedInfoProp('phoneModel', 'PROFILE_PHONE_MODEL')
   if global._CC.config.wishlist.table
     .box
-      table.table.has-mobile-cards
+      table.table.has-mobile-cards#wishlist-table
         thead
           th #
           th(style='width: 15%;')= lang('WISHLIST_IMAGE')
           th(style='width: 25%;')= lang('WISHLIST_NAME')
           th(style='width: 50%;')= lang('WISHLIST_NOTE')
-          th(style='width: 10%;')= lang('WISHLIST_PRICE')
+          th(style='width: 10%;')
+            a(href="#", id="sort-price")= lang('WISHLIST_PRICE')
```

Comment:
I think the a should contain the th, so you can click the entire box rather than just the text.

Reply:
I cannot for the life of me get your requested behavior to work. How important is this to you?

Reply:
Not super important, I'll give it a stab but it's fine if we don't have it